tools/validators/instance_validator/validate/entity_instance.py

```diff
@@ -342,9 +363,11 @@ def _ValidateTranslation(self, entity: EntityInstance, is_udmi: bool = False
       unmatched = set(type_fields.keys()).difference(set(found_fields.keys()))
       for unmatched_name in unmatched:
         if not type_fields[unmatched_name].optional:
-          print(f'[ERROR]\tEntity {entity.guid} ({entity.code}) missing field '
-                f'"{unmatched_name}" which is required for assigned type '
-                f'"{entity.type_name}"')
+          print(
```

Comment:
same here, the autoformatting chaos?

Reply:
Yeah, this has to stay to avoid line-too-long errors.